utils/network.py

```diff
@@ -0,0 +1,107 @@
+import networkx as nx
+import pandas as pd
+
+from utils.linkage import deduplicate_perfect_matches
+
+
+def deduplicate_datasets(
+    ind_df: pd.DataFrame, org_df: pd.DataFrame, transactions_df: pd.DataFrame
+) -> tuple:
+    """Deduplicates the uuids in the inds and orgs dfs and updates the uuids in
+    transactions dataset to match those in the new inds and orgs dfs
+
+    Args:
+        ind_df: A pandas df with individual information
+        org_df: A pandas df with organization information
+        transactions df: A pandas df with info on transactions between entities
+
+    Returns:
+        A tuple of the ind_df, org_df, and transactions_df
+    """
+    # apply dedup to both inds and orgs
+    inds_df = deduplicate_perfect_matches(ind_df)
+    orgs_df = deduplicate_perfect_matches(org_df)
+
+    # update the deduplicated uuids in transaction donor and recipient columns
+    # to the uuids they are mapped to
+    deduped = pd.read_csv("../output/deduplicated_UUIDs.csv")
+    transactions_df[["donor_id", "recipient_id"]] = transactions_df[
+        ["donor_id", "recipient_id"]
+    ].replace(deduped)
+
+    return inds_df, orgs_df, transactions_df
+
+
+def name_identifier(uuid: str, orgs_df, inds_df) -> str:
```

Comment:
I don't think this needs to be specific to ords/inds. What you could do is change the arguments on the function to take in UUID, and a list of DFs and then iterate through the list until the ID is found and then return it, if not then ... -> need to specific that there is an expectation that the dfs in the list have both an 'id' column and a 'name' column